sys/net/gnrc/Makefile.dep

```diff
@@ -493,5 +493,6 @@ ifneq (,$(filter gnrc_lwmac,$(USEMODULE)))
   USEMODULE += gnrc_netif
   USEMODULE += gnrc_nettype_lwmac
   USEMODULE += gnrc_mac
+  USEMODULE += xtimer
```

Comment:
Sneaked in? Otherwise, lgtm

Reply:
Nope. gnrc_lwmac needs xtimer. But it never made it into the dependency list ;-)
Compile tests failed after at86rf2xx hasn't required xtimer anymore.

Reply:
OK, sorry for the noise